README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
- [ ] Kindly edit your readme to be more detailed and descriptive using the template from microverse